host/sgx/sgxquoteprovider.c

```diff
@@ -50,6 +50,121 @@ oe_result_t oe_initialize_quote_provider()
     return result;
 }
 
+// oe_result_t
```

Comment:
Is this line (and the one below) intended?

Reply:
Thanks, the line below removed.